src/main/java/seedu/address/logic/commands/OpenReportCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.hirelah.Interviewee;
+import seedu.address.model.hirelah.IntervieweeList;
+import seedu.address.model.hirelah.exceptions.IllegalActionException;
+
+/**
+ * OpenReportCommand describes the behavior when the
+ * client wants to open an interview report of an interviewee.
+ */
+public class OpenReportCommand extends Command {
+    public static final String COMMAND_WORD = "report";
+    public static final String MESSAGE_SUCCESS = "Interview report opened: %1$s";
+    public static final String MESSAGE_USAGE = "open " + COMMAND_WORD
+            + ": Opens the interview report of an interviewee."
+            + "Example: list " + COMMAND_WORD + " John Doe";
+
```

Comment:
Uh, so is the command word "report", "list" or "open"? According to the Parser, it is `open <identifier>`, but here it seems to be `open report`, but in the Example on line 20, it says `list report John Doe`